central/hash/manager/deduper.go

```diff
@@ -24,6 +24,8 @@ type Deduper interface {
 	// MarkSuccessful marks the message if necessary as being successfully processed, so it can be committed to the database
 	// It will promote the message from the received map to the successfully processed map
 	MarkSuccessful(msg *central.MsgFromSensor)
+	// MarkUnsuccessful deletes the msg from the received map in order to allow future messages a chance to be successfully processed
+	MarkUnsuccessful(msg *central.MsgFromSensor)
```

Comment:
Nit: Are we actually marking the message? Otherwise, how about renaming this to `DropMessage`?
(This also applies to `MarkSuccessful`, which is existing code).

Reply:
Changed to RemoveMessage